web/src/main/java/org/akaza/openclinica/service/StudyParticipantServiceImpl.java

```diff
@@ -99,19 +88,19 @@
     public static final String EMAIL_PATTERN = "^[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[A-Za-z0-9](?:[A-Za-z0-9-]*[A-Za-z0-9])?\\.)+[A-Za-z0-9](?:[A-Za-z0-9-]*[A-Za-z0-9])?$";
 
 
-
+    private  StudyBean currentStudy;
+    private  StudyBean siteStudy;
```

Comment:
why do we need to add these as global fields? Can they be passed to isEnrollmentCapped() method as arguments?